sdk/src/main/java/io/radar/sdk/RadarJobScheduler.kt

```diff
@@ -40,21 +48,29 @@ class RadarJobScheduler : JobService() {
                 putString(EXTRA_SOURCE, source.name)
             }
 
-            val jobInfo = JobInfo.Builder(JOB_ID, componentName)
+            val settings = RadarSettings.getFeatureSettings(context)
+            val jobId = if (counter.get() < settings.maxConcurrentJobs) {
+                BASE_JOB_ID + counter.incrementAndGet()
+            } else {
+                // Replace a random job
+                BASE_JOB_ID + Random.nextInt(settings.maxConcurrentJobs)
+            }
+            val jobInfo = JobInfo.Builder(jobId, componentName)
```

Comment:
@tjulien this technically does work, however the logic for replacing is not as good, since it would always replace the most-recently-scheduled job, not the first-scheduled job. The current solution is not optimized either, but may be slightly improved.

Reply:
I suspect we can just set counter = 0 instead of decrementing

Reply:
ok, done